app/views/@shared/_page_heading.html.erb

```diff
@@ -129,11 +133,23 @@
                   </li>
                   <!-- Accessibility -->
 
-                  <!-- Others -->
-                  <li class="nav-item u-header__nav-item">
-                    <a class="nav-link u-header__nav-link" href="plans.html">Plans</a>
-                  </li>
-                  <!-- Others -->
+                  <% if current_user.administrator? %>
```

Comment:
We'll likely want to swap these types of checks out for the `authorizable` pattern on the images branch after it merges.